README.md

```diff
@@ -0,0 +1,169 @@
+# Lecture 05 otus Introduction with GCP (via KodeKloud Playground)
+
+* Create user account in GCP (use KodeKloud Playground)
+* Create instances VM in web-interface and connecting to they via SSH
+* Consider the options for connecting to hosts via bastion host and VPN
+
+## Create new branch cloud-bastion in github repositor (devops_study_infra)
+
+git checkout -b cloud-bastion
+
+## Login to GCP
+
+1. Login to KodeKloude.
+
+2. Select Playground.
+
+3. Create privat tab in browser.
```

Comment:
Here a mistyping `privat` should be `private`

JFY: I advice you to use spell checker because it reduce errors count not only in documentation but in code too (especially in languages with weak type control)

I use this one https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker